source/custom-output-cli.txt

```diff
@@ -0,0 +1,250 @@
+.. _go-template-output: 
+
+=============================
+Customize the {+atlas-cli+} Output
+=============================
+
+.. default-domain:: mongodb
+
+You can customize the {+atlas-cli+} output fields and format using a ``Go`` 
+template or a JSON path.
+                    
+Go templates
+------------
+                    
+You can specify the Go template with the command for a simple output 
+or through a separate file for a complex output. For information on the 
+template, see `Package template <https://golang.org/pkg/text/template/>`__. 
+For information on the types and properties available for each response, see: 
+
+- `Atlas types <https://go.mongodb.org/atlas/mongodbatlas/>`__
```

Comment:
Removing the bullet since the other list items are now gone
```suggestion
To learn the types and properties available for each response, see 
`Atlas types <https://go.mongodb.org/atlas/mongodbatlas/>`__.
```